docs/data/material/components/autocomplete/Virtualize.js

```diff
@@ -123,16 +122,6 @@ function random(length) {
   return result;
 }
 
-const StyledPopper = styled(Popper)({
```

Comment:
How is this change relevant to the PR?

Reply:
It wouldn't have worked, it needs padding: 0 and margin: 0, it would break without:

<img width="322" alt="SCR-20241111-tpxu" src="https://github.com/user-attachments/assets/790c9412-63b2-453a-b878-2c23a96ab492">